mcs_benchmark_data/loaders/rdf_file_loader.py

```diff
@@ -29,11 +30,11 @@ def _flush(self, models):
         context = [
             "https://tetherless-world.github.io/mcs-ontology/utils/context.jsonld",
```

Comment:
Make context a default parameter of the loader, please.

Since dicts are mutable the usual pattern is 

```
class X:
    SOME_PARAM_DEFAULT = [..., {...}]

    def __init__(self, *, some_param=None):
        if some_param is None:
            self.__some_param = self.SOME_PARAM_DEFAULT
```